doc/contributing/submitting-changes.xml

```diff
@@ -305,6 +305,11 @@ Additional information.
     review uncommitted changes:
 <screen>nix-shell -p nix-review --run "nix-review wip"</screen>
    </para>
+
+   <para>
+    review changes from last commit:
+<screen>nix-shell -p nix-review --run "nix-review rev HEAD"</screen>
```

Comment:
This could be even switched to the new `nix run`:

```suggestion
<screen>nix run nixpkgs.nix-review -c nix-review rev HEAD</screen>
```

This avoids pulling some dependencies/environment variables that `nix-shell` would do

Reply:
Then for all three list items:
```suggestion
   <para>
    review changes from pull request number 12345:
<screen>run nixpkgs.nix-review -c nix-review pr 12345</screen>
   </para>

   <para>
    review uncommitted changes:
<screen>run nixpkgs.nix-review -c nix-review wip</screen>
   </para>

   <para>
    review changes from last commit:
<screen>run nixpkgs.nix-review -c nix-review rev HEAD</screen>
   </para>
```

Reply:
> Then for all three list items:

Applied in commit https://github.com/NixOS/nixpkgs/pull/73217/commits/e4c26a9a205df348b969e97c1c7db873be51dfcf